### DISCUSSION TITLE
Menu child components onFocus firing on hover

### DISCUSSION THREAD

**Discussion by veloware** [chakra-ui/chakra-ui#8655] [General]
### Description

When I click on a `MenuButton`  and then `MenuList` and `MenuItem`s appear, when I hover over any of the `MenuItem`s, their `onFocus` handler gets called, and also the `onFocus` of the `MenuList`

Example code to reproduce - 

```
<Menu>
  <MenuButton as={Button} rightIcon={<ChevronDownIcon />} onFocus={(e) => {console.log('MenuButton')}}>
    Actions
  </MenuButton>
  <MenuList onFocus={(e) => {console.log('MenuList')}}>
    <MenuItem onFocus={(e) => {console.log('MenuItem - download')}}>Download</MenuItem>
    <MenuItem>Create a Copy</MenuItem>
    <MenuItem>Mark as Draft</MenuItem>
    <MenuItem>Delete</MenuItem>
    <MenuItem>Attend a Workshop</MenuItem>
  </MenuList>
</Menu>
```

and was reproduced here by adding the above `console.log()` statements - https://v2.chakra-ui.com/docs/components/menu/usage#usage

I wouldn't expect `onFocus` to be fired on any of the child components unless they received focus, and I definitely wouldn't expect the `onFocus` to keep getting invoked as I hover over the MenuItems

### Link to Reproduction

https://v2.chakra-ui.com/docs/components/menu/usage#usage

### Steps to reproduce

See description

### Chakra UI Version

2.8.2

### Browser

Chrome 126.0.6478.115

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by segunadebayo** [comment]
This is by design. The menu item is focused on both keyboard and pointer interactions.

In v3 is this redesigned to avoid triggering the `onFocus` event

**Reply by veloware** [reply]
thanks for getting back, at least I'm not just going mad :)

**Comment by segunadebayo** [comment]
v2 focuses the item on pointer hover. That is by design. In v3, style `_highlighted` instead.
